docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
     restart: on-failure
     depends_on:
       - db
-    image: vulcanize/statediff-migrations:v0.3.0
+    image: vulcanize/statediff-migrations:v0.4.0
```

Comment:
Check if this is the latest one?

Reply:
Updated the version.

Reply:
The test fails after upgrading the docker images.